slack_bolt/util/utils.py

```diff
@@ -88,3 +89,14 @@ def get_name_for_callable(func: Callable) -> str:
 
 def get_arg_names_of_callable(func: Callable) -> List[str]:
     return inspect.getfullargspec(inspect.unwrap(func)).args
+
+
+def extract_callables(
```

Comment:
I think that this new name is too general. The list of callable that this method returns is only useable as listener functions. How about naming to `extract_listener_callables` or `extract_listener_functions`?

Reply:
Agreed, changed to `extract_listener_callables`